src/Functions/substring.cpp

```diff
@@ -31,40 +31,40 @@ namespace
 {
 
 /// If 'is_utf8' - measure offset and length in code points instead of bytes.
-/// UTF8 variant is not available for FixedString arguments.
 template <bool is_utf8>
 class FunctionSubstring : public IFunction
 {
 public:
     static constexpr auto name = is_utf8 ? "substringUTF8" : "substring";
-    static FunctionPtr create(ContextPtr)
-    {
-        return std::make_shared<FunctionSubstring>();
-    }
-
-    String getName() const override
-    {
-        return name;
-    }
 
+    static FunctionPtr create(ContextPtr) { return std::make_shared<FunctionSubstring>(); }
+    String getName() const override { return name; }
```

Comment:
here is it a one-liner
```C++
String getName() const override { return name; }
```
however, you changed it in EnumSource like:
```C++
size_t rowNum() const
{
    return row_num;
}
```
?

Reply:
I changed it in EnumSource to achieve consistent formatting with StringSource which is in the same file (Sources.h).

For functions, the one-liner approach is okay.